### PR TITLE
feat: added Home page

### PR DESCRIPTION
**Pull request by Priya-Patel31**
It consists of :
* Header
* Main-section
 • Sidebar
 • Main-container

And made it fully responsive .

Preview Link : 

https://vibrant-babbage-d00102.netlify.app/

### CODE REVIEW

**Review by Manju-sree — CHANGES_REQUESTED** [review]
please add preview links and you are directly merging into main from dev.. please check

**Comment by Priya-Patel31** [comment]
> please add preview links and you are directly merging into main from dev.. please check

Already added in comment !!!

**Comment by Priya-Patel31** [comment]
> please add preview links and you are directly merging into main from dev.. please check

Just to raise a PR but later I won't merge dev into main branch

**Review by Manju-sree — APPROVED** [review]
very well structured code.. its good to merge now

File: index.css [hunk 1]
```diff
@@ -0,0 +1,210 @@
+.homePage-container {
```
Comment (JantuDeb) [inline index.css:1]: Avoid mix naming convention for class names `homepage-container` or `home-page-container` would be good

File: index.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        <div class="search-bar-container">
+          <button type="submit">
+            <i class="fas fa-search"></i>
+          </button>
+          <input type="search" placeholder="Search" class="search-bar" />
+        </div>
+
+        <div class="flex-row justify-center align-center">
+          <div class="mr-2"><a href="/authentication/signin/signin.html">Login</a></div>
+          <div class="component-image-rounded user-sm-image">
```
Comment (JantuDeb) [inline index.html:33]: This class name is too long,  `image-rounded` is fine I guess.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                alt="user-image"
+              />
+            </div>
+            <div class="user-details-container">
+              <div class="user-details-section1-container">
+                <h2>Priya Patel</h2>
+                <small>priya@gmail.com</small>
+              </div>
+              <div class="user-details-section2-container mt-2 flex-row">
+                <div class="total-score-container flex-row">
```
Comment (t007rushi) [inline index.html]: you can have consistent inside wrap/container in naming as here user-total-score instead total-score

File: index.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="main-content-section2-container p-1 flex-row">
+            <div class="main-content-section2-card-container">
+              <h2 class="py-2 mb-2 flex-row justify-center">Achievements</h2>
+              <div class="user-achievements-container flex-col">
+                <div class="flex-row justify-around p-2">
+                  <div class="trophy-container component-image-square">
```
Comment (t007rushi) [inline index.html:108]: same as above

File: index.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+                <div class="space-1rem"></div>
+                <div class="trophy-container component-image-square">
+                  <img src="/images/medal (3).png" alt="medal-image" />
+                  <p>3rd Prize</p>
+                </div>
+                <div class="space-1rem"></div>
+              </div>
+            </div>
+            <div class="main-content-section2-card-container">
+              <h2 class="py-2 flex-row justify-center">Featured Category</h2>
```
Comment (t007rushi) [inline index.html:126]: py-2?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (t007rushi) [inline index.html]: Quizbuzz is missing

**Review by t007rushi — APPROVED** [review]
Small changes but UI with dark mode so looking clean

File: index.html [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+                  <div class="flex-col align-center">
+                    <h2 class="">200</h2>
+                    <p>Answers</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="main-content-section2-container p-1 flex-row">
+            <div class="main-content-section2-card-container">
```
Comment (JantuDeb) [inline index.html:104]: Same as above too long class names.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+                </div>
+                <div class="space-1rem"></div>
+                <div class="trophy-container component-image-square">
+                  <img src="/images/medal (3).png" alt="medal-image" />
+                  <p>3rd Prize</p>
+                </div>
+                <div class="space-1rem"></div>
+              </div>
+            </div>
+            <div class="main-content-section2-card-container">
```
Comment (JantuDeb) [inline index.html:125]: Classname is too long

**Review by JantuDeb — APPROVED** [review]
Suggested some changes, these class names are not descriptive and too long

File: index.html (hunk 6 above)
Comment (Priya-Patel31) [reply index.html]: changed it!!

File: index.css (hunk 1 above)
Comment (Priya-Patel31) [reply index.css:1]: you mean to avoid camel case naming convention??

File: index.html (hunk 2 above)
Comment (Priya-Patel31) [reply index.html:33]: i'll try to name  little shorter

File: index.html (hunk 7 above)
Comment (Priya-Patel31) [reply index.html:104]: But if this className is making sense then we can give bigger name to it as I have already given main-content-container to one of the div's hence can't use that so used this . 
And we have to take care of better naming convention too hence used as it is making sense by the name itself.

File: index.html (hunk 8 above)
Comment (Priya-Patel31) [reply index.html:125]: Same as above !!

File: index.html (hunk 3 above)
Comment (Priya-Patel31) [reply index.html]: changed it to score-container

File: index.css (hunk 1 above)
Comment (JantuDeb) [reply index.css:1]: > you mean to avoid camel case naming convention??

Yes, for classes don't use camel case

File: index.html (hunk 7 above)
Comment (JantuDeb) [reply index.html:104]: Use classes from component library whenever possible

File: index.html (hunk 5 above)
Comment (Priya-Patel31) [reply index.html:126]: it stands for padding along y-axis if still doubt persists go through tailwind CSS .

**Review by Ankur9669 — APPROVED** [review]
Great Code looks good to merge

**Pull request closed without merging**